examples/asr/conf/carnelinet/carnelinet_384.yaml

```diff
@@ -0,0 +1,259 @@
+
+      - filters: 384
+        repeat: ${model.model_defaults.repeat}
+        kernel: ["${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}"]
+        stride: [1]
+        dilation: [1]
+        dropout: ${model.model_defaults.dropout}
+        residual: true
+        separable: ${model.model_defaults.separable}
+        se: ${model.model_defaults.se}
+        se_context_size: ${model.model_defaults.se_context_size}
+        se_repeat: ${model.model_defaults.se_repeat}
+
+      - filters: 384
+        repeat: ${model.model_defaults.repeat}
+        kernel: [7]
+        stride: [2]  # *stride
+        dilation: [1]
+        dropout: ${model.model_defaults.dropout}
+        residual: true
+        separable: ${model.model_defaults.separable}
+        se: ${model.model_defaults.se}
+        se_context_size: ${model.model_defaults.se_context_size}
+        se_repeat: ${model.model_defaults.se_repeat}
+        stride_last: true
+        residual_mode: "stride_add"
+
+      - filters: 384
+        repeat: ${model.model_defaults.repeat}
+        kernel: ["${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}"]
+        stride: [1]
+        dilation: [1]
+        dropout: ${model.model_defaults.dropout}
+        residual: true
+        separable: ${model.model_defaults.separable}
+        se: ${model.model_defaults.se}
+        se_context_size: ${model.model_defaults.se_context_size}
+        se_repeat: ${model.model_defaults.se_repeat}
+
+      - filters: 384
+        repeat: ${model.model_defaults.repeat}
+        kernel: [13]
+        stride: [2]  # stride
+        dilation: [1]
+        dropout: ${model.model_defaults.dropout}
+        residual: true
+        separable: ${model.model_defaults.separable}
+        se: ${model.model_defaults.se}
+        se_context_size: ${model.model_defaults.se_context_size}
+        se_repeat: ${model.model_defaults.se_repeat}
+        stride_last: true
+        residual_mode: "stride_add"
+
+      - filters: 384
+        repeat: ${model.model_defaults.repeat}
+        kernel: ["${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}"]
+        stride: [1]
+        dilation: [1]
+        dropout: ${model.model_defaults.dropout}
+        residual: true
+        separable: ${model.model_defaults.separable}
+        se: ${model.model_defaults.se}
+        se_context_size: ${model.model_defaults.se_context_size}
+        se_repeat: ${model.model_defaults.se_repeat}
+
+      - filters: &enc_final 640
```

Comment:
This can be in model_defaulfs.enc_hidden

Reply:
Chose a different name.